Stronger/StrongerDelegate.swift

```diff
@@ -25,10 +25,10 @@ class StrongerDelegate: SpeziAppDelegate {
                 AccountConfiguration(configuration: [
                     .requires(\.userId),
                     .requires(\.name),
-
-                    // additional values stored using the `FirestoreAccountStorage` within our Standard implementation
-                    .collects(\.genderIdentity),
-                    .collects(\.dateOfBirth)
+                    .requires(\.password),
```

Comment:
This line is actually not necessary and will cause problems if you use Single-Sign-On Account Services like Sign in with Apple that don't require a password. SpeziAccount will make sure that the password field is shown for account services that require a password as their credentials. That's just a small technical detail. We try to improve that documentation around this 👍